core/src/main/java/org/infinispan/interceptors/impl/ActivationInterceptor.java

```diff
@@ -1,21 +1,22 @@
-package org.infinispan.interceptors;
+package org.infinispan.interceptors.impl;
 
 import org.infinispan.commands.FlagAffectedCommand;
 import org.infinispan.commands.write.PutMapCommand;
 import org.infinispan.context.InvocationContext;
 import org.infinispan.util.logging.Log;
```

Comment:
Sorry, I hope we're going to have a checkstyle rule for unused imports soon, then I'll just have to remember to run the build ;)